doc/whats_new/v1.2.rst

```diff
@@ -232,6 +232,11 @@ Changelog
   of a binary classification problem. :pr:`22518` by
   :user:`Arturo Amor <ArturoAmorQ>`.
 
+- |Fix| Allows `csr_matrix` as input for parameter: `y_true` of
+   the :func:`metrics.label_ranking_average_precision_score` metric.
+   :pr:`23442` by :user:`Sean Atukorala <ShehanAT>`
+  :pr:`23442` by :user:`Sean Atukorala <ShehanAT>`
```

Comment:
```suggestion
```
This line should be removed. It generates a sphinx error preventing the documentation from being built.